lib/potassium/cli_options.rb

```diff
@@ -121,12 +121,26 @@ module Potassium::CliOptions # rubocop:disable Metrics/ModuleLength
     },
     {
       type: :switch,
-      name: "github-private",
-      desc: "Whether to the github repository is private",
+      name: "github_private",
+      desc: "Whether the github repository is private or not",
       negatable: true,
       default_value: false,
       default_test_value: false
     },
+    {
+      type: :flag,
```

Comment:
cuando es un input también se llama flag?

Reply:
Entiendo que sí: https://github.com/davetron5000/gli/wiki/Declaring-flags-and-switches